dcrwallet.go

```diff
@@ -189,9 +190,10 @@ func walletMain() error {
 }
 
 // startPromptPass prompts the user for a password to unlock their wallet in
-// the event that it was restored from seed or --promptpass flag is set.
+// the event that it was restored from seed or --promptpass/enablestakemining
+// flag is set.
 func startPromptPass(w *wallet.Wallet) {
-	promptPass := cfg.PromptPass
+	promptPass := cfg.PromptPass || cfg.EnableStakeMining
```

Comment:
on second thought this is probably a bad idea since it will break scripts.  Probably want to just continue requiring the user to manually unlock the wallet without timeout.